views/views.go

```diff
@@ -89,12 +139,18 @@ type (
 var _ Repo = &Views{}
 
 // New initialises connections, templates, and cookies
-func New(conf Config) *Views {
-	v := Views{}
+func New(conf *Config, host string) *Views {
+	v := &Views{}
 	// Connecting to stores
 	dbStore, err := db.NewStore(conf.DatabaseURL)
 	if err != nil {
-		log.Fatalf("NewStore failed: %+v", err)
+		if conf.Debug {
+			log.Printf("db failed: %+v", err)
```

Comment:
Does it make sense to continue, even in development, without a DB connection? Are there any features that are useful without one?

Reply:
In debug I may just be testing a page layout, this is so it can have offline development or requires a lot of hoops to jump through